reverse_array.rb

```diff
@@ -0,0 +1,21 @@
+=begin
+5. Write a function to reverse an array. (WITHOUT using the built in reverse function). Write your algorithm to create your own new array. 
+reverse([1, 2, 3, 4]) ➞ [4, 3, 2, 1]
+reverse([9, 9, 2, 3, 4]) ➞ [4, 3, 2, 9, 9]
+reverse([]) ➞ []
+=end
+
+def reverse(array)
+   return "Empty array" if array.empty? 
+  reversed = Array.new
+  (1..array.size).each do |i|
+    reversed << array[i * -1]
+  end
+  "Array being reversed to #{reversed}"

```

Comment:
careful on what you the method should return. 

methods should return data that can be later used by where they are getting called from. don't format responses. 

also be sure to look at what the question asks the return to be .

Reply:
Do you mean this extra  string to the output 
"Array being reversed to #{reversed}"

Reply:
yup. So instead let the method's return always be just the data. So return just the array instead of strings for this question because that is what the question is expecting.

Reply:
Oh i got you.
 fixed it